### PR TITLE
Makes enviroskirts linked to the suitskirt pref

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
kill me

## Why It's Good For The Game
is anything anymore

## Changelog
:cl:
tweak: enviroskirts are now linked to the suitskirt preference
/:cl:

### CODE REVIEW

**Comment by dinnerwoerror** [comment]
hold it

you mean i coulda just added a fuckin forward slash the first time and we wouldn't be in this situation right now

**Comment by dee-dubya** [comment]
> 
> 
> hold it
> 
> you mean i coulda just added a fuckin forward slash the first time and we wouldn't be in this situation right now

welcome to coding

**Comment by dinnerwoerror** [comment]
well in that case i guess it means we can appease critatard and cut the envirosuits from the clothing vendors as well

**Comment by Maldaris** [comment]
> well in that case i guess it means we can appease critatard and cut the envirosuits from the clothing vendors as well

we should do this in the same PR

**Comment by Superyodeler** [comment]
@MarkSuckerberg do you want me to review this now, or do you want remove the envirosuits in this pr?

**Comment by MarkSuckerberg** [comment]
I'll remove em, yeah, give me a sec

**Comment by MarkSuckerberg** [comment]
Scratch that, I'll remove that in a different PR if needed.

**Comment by Superyodeler** [comment]
I'll review this tomorrow

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 1]
```diff
@@ -160,6 +160,19 @@
 		if("Clown")
 			O = new /datum/outfit/plasmaman/clown
 
+	var/holder
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm:163]:
```suggestion
	var/holder		// Wasp Edit Begin - Plasma skirtsuit prefs
 ```

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 2]
```diff
@@ -160,6 +160,19 @@
...
+		if(PREF_SKIRT)
+			holder = "[O.uniform]/skirt"
+		if(PREF_GREYSUIT)
+			O.head = /obj/item/clothing/head/helmet/space/plasmaman
+			holder = "/obj/item/clothing/under/plasmaman"
+		else
+			holder = "[O.uniform]"
+
+	if(text2path(holder))
+		O.uniform = text2path(holder)
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm:174]:
```suggestion
		O.uniform = text2path(holder)		// Wasp Edit End
```

**Comment by MarkSuckerberg** [comment]
when

**Comment by MarkSuckerberg** [comment]
when

**Comment by MarkSuckerberg** [comment]
when

**Comment by Superyodeler** [comment]
I've approved

**Comment by Superyodeler** [comment]
When you

**Pull request merged**